fastplotlib/layouts/_gridplot.py

```diff
@@ -13,12 +13,16 @@
 
 if is_jupyter():
     from ipywidgets import HBox, Layout, Button, ToggleButton, VBox, Dropdown
+    from sidecar import Sidecar
+    from IPython.display import display
 
 from ._utils import make_canvas_and_renderer
 from ._defaults import create_controller
 from ._subplot import Subplot
 from ._record_mixin import RecordMixin
 
+PLOT_OPEN = False
```

Comment:
This cannot be a global because it would then be shared across plot instances. It must be an attribute of the plot instance.